Can't run more than one program at the same time from SBT console

To reproduce:

```
sbt:knora> webapi / reStart
sbt:knora> webapi / runMain org.knora.webapi.util.TransformOntology
```

Error:

```
[error] Error: Exception thrown by the agent : java.rmi.server.ExportException: Port already in use: 1617; nested exception is: 
[error] 	java.net.BindException: Address already in use (Bind failed)
[error] jdk.internal.agent.AgentConfigurationError: java.rmi.server.ExportException: Port already in use: 1617; nested exception is: 
[error] 	java.net.BindException: Address already in use (Bind failed)
[error] 	at jdk.management.agent/sun.management.jmxremote.ConnectorBootstrap.startRemoteConnectorServer(ConnectorBootstrap.java:491)
[error] 	at jdk.management.agent/jdk.internal.agent.Agent.startAgent(Agent.java:450)
[error] 	at jdk.management.agent/jdk.internal.agent.Agent.startAgent(Agent.java:621)
[error] Caused by: java.rmi.server.ExportException: Port already in use: 1617; nested exception is: 
[error] 	java.net.BindException: Address already in use (Bind failed)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport.listen(TCPTransport.java:335)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport.exportObject(TCPTransport.java:243)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.exportObject(TCPEndpoint.java:411)
[error] 	at java.rmi/sun.rmi.transport.LiveRef.exportObject(LiveRef.java:147)
[error] 	at java.rmi/sun.rmi.server.UnicastServerRef.exportObject(UnicastServerRef.java:233)
[error] 	at java.rmi/sun.rmi.registry.RegistryImpl.setup(RegistryImpl.java:220)
[error] 	at java.rmi/sun.rmi.registry.RegistryImpl.<init>(RegistryImpl.java:180)
[error] 	at jdk.management.agent/sun.management.jmxremote.SingleEntryRegistry.<init>(SingleEntryRegistry.java:49)
[error] 	at jdk.management.agent/sun.management.jmxremote.ConnectorBootstrap.exportMBeanServer(ConnectorBootstrap.java:836)
[error] 	at jdk.management.agent/sun.management.jmxremote.ConnectorBootstrap.startRemoteConnectorServer(ConnectorBootstrap.java:479)
[error] 	... 2 more
[error] Caused by: java.net.BindException: Address already in use (Bind failed)
[error] 	at java.base/java.net.PlainSocketImpl.socketBind(Native Method)
[error] 	at java.base/java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:437)
[error] 	at java.base/java.net.ServerSocket.bind(ServerSocket.java:381)
[error] 	at java.base/java.net.ServerSocket.<init>(ServerSocket.java:243)
[error] 	at java.base/java.net.ServerSocket.<init>(ServerSocket.java:135)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPDirectSocketFactory.createServerSocket(TCPDirectSocketFactory.java:45)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newServerSocket(TCPEndpoint.java:666)
[error] 	at java.rmi/sun.rmi.transport.tcp.TCPTransport.listen(TCPTransport.java:324)
```

I've tried this with commits to `develop` since release 7.0.0, and found that the problem was introduced with PR #1201.

--- a/KnoraBuild.sbt
+++ b/KnoraBuild.sbt
@@ -335,12 +335,12 @@ lazy val webapi = knoraModule("webapi")
 
             fork := true, // always fork
 
-            run / javaOptions := webapiJavaRunOptions,
+            // run / javaOptions := webapiJavaRunOptions,
 
             reStart / javaOptions ++= resolvedJavaAgents.value map { resolved =>
                 "-javaagent:" + resolved.artifact.absolutePath + resolved.agent.arguments
             }, // allows sbt-javaagent to work with sbt-revolver
-            reStart / javaOptions ++= webapiJavaRunOptions,
+            // reStart / javaOptions ++= webapiJavaRunOptions,
 
             Test / parallelExecution := false,
             Test / javaOptions ++= Seq("-Dconfig.resource=graphdb-se.conf") ++ webapiJavaTestOptions,
@@ -434,6 +434,7 @@ lazy val webapi = knoraModule("webapi")
             buildInfoPackage := "org.knora.webapi"
         )
 
+/*
 lazy val webapiJavaRunOptions = Seq(
     // "-showversion",
     "-Xms1G",
@@ -447,6 +448,7 @@ lazy val webapiJavaRunOptions = Seq(
     "-Dcom.sun.management.jmxremote.ssl=false",
     //"-agentpath:/Applications/YourKit-Java-Profiler-2018.04.app/Contents/Resources/bin/mac/libyjpagent.jnilib"
 )
+*/
 
 lazy val webapiJavaTestOptions = Seq(
     // "-showversion",

--- a/KnoraBuild.sbt
+++ b/KnoraBuild.sbt
@@ -335,12 +335,12 @@ lazy val webapi = knoraModule("webapi")
 
             fork := true, // always fork
 
-            // run / javaOptions := webapiJavaRunOptions,
+            run / javaOptions := webapiJavaRunOptions,
 
             reStart / javaOptions ++= resolvedJavaAgents.value map { resolved =>
                 "-javaagent:" + resolved.artifact.absolutePath + resolved.agent.arguments
             }, // allows sbt-javaagent to work with sbt-revolver
-            // reStart / javaOptions ++= webapiJavaRunOptions,
+            reStart / javaOptions ++= webapiJavaRunOptions,
 
             Test / parallelExecution := false,
             Test / javaOptions ++= Seq("-Dconfig.resource=graphdb-se.conf") ++ webapiJavaTestOptions,
@@ -434,7 +434,6 @@ lazy val webapi = knoraModule("webapi")
             buildInfoPackage := "org.knora.webapi"
         )
 
-/*
 lazy val webapiJavaRunOptions = Seq(
     // "-showversion",
     "-Xms1G",
@@ -443,12 +442,11 @@ lazy val webapiJavaRunOptions = Seq(
     //"-XX:+UseG1GC",
     //"-XX:MaxGCPauseMillis=500"
     "-Dcom.sun.management.jmxremote",
-    "-Dcom.sun.management.jmxremote.port=1617",
+    // "-Dcom.sun.management.jmxremote.port=1617",
     "-Dcom.sun.management.jmxremote.authenticate=false",
     "-Dcom.sun.management.jmxremote.ssl=false",
     //"-agentpath:/Applications/YourKit-Java-Profiler-2018.04.app/Contents/Resources/bin/mac/libyjpagent.jnilib"
 )
-*/
 
 lazy val webapiJavaTestOptions = Seq(
     // "-showversion",

